coordinator/performance/monitor.go

```diff
@@ -0,0 +1,79 @@
+package performance
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/mattermost/mattermost-load-test-ng/coordinator/performance/prometheus"
+
+	"github.com/mattermost/mattermost-server/v5/mlog"
+)
+
+type Monitor struct {
+	config     MonitorConfig
+	helper     *prometheus.Helper
+	stopChan   chan struct{}
+	statusChan chan Status
+}
+
+func NewMonitor(config MonitorConfig) (*Monitor, error) {
+	if ok, err := config.IsValid(); !ok {
+		return nil, err
+	}
+	helper, err := prometheus.NewHelper(config.PrometheusURL)
```

Comment:
The word "helper" sounds odd to me. From the code, it seems like it wraps a subset of the prometheus APIs. How about `NewWrapper` ? And respectively changing all helpers to wrappers.